package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "13-mongo_single_resource_api",
+  "version": "1.0.0",
+  "description": "![CF](https://camo.githubusercontent.com/70edab54bba80edb7493cad3135e9606781cbb6b/687474703a2f2f692e696d6775722e636f6d2f377635415363382e706e67) Lab 13: Single Resource Mongo and Express API ===",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "start": "node server.js",
+    "test": "DEBUG='13-mongo_single_resource_api*' mocha"
```

Comment:
This isn't quite right for running your debugger.  Check out the way we did it in class.